src/components/Common/SearchInput.tsx

```diff
@@ -106,9 +110,60 @@ const SearchInputFieldRenderer = ({
             ref={inputRef}
             {...prop}
           />
-          {!isSingleOption && <KeyboardShortcutHint open={open} />}
+          {!isSingleOption && !isMobile && <KeyboardShortcutHint open={open} />}
+        </div>
+      );
+
+    case "aadhar_card":
```

Comment:
No, you shouldn't be writing specific logic like this in the first place, because this is just a sample config.

Look at patient_instance_identifier_configs object and what attributes it has. You need to modify PatientIndex and SearchInput accordingly.